pkg/apis/node/types.go

```diff
@@ -47,13 +48,13 @@ type RuntimeClass struct {
 	// immutable.
 	Handler string
```

Comment:
@ruquanzhao Thank you for your PR. Can you explain this type of changes, please? Why do you prefer to use lowercased names for capitalised fields?

Reply:
This is about issue #113438.

In a word, using lowercased names will make the generated docs and field names match in https://github.com/kubernetes/kubernetes/blob/master/api/openapi-spec/swagger.json

Is this make sense to you?

Reply:
Thanks for the explanations.
/lgtm